src/NodeRTLib/ProjectTemplates/binding.gyp

```diff
@@ -51,7 +51,9 @@
           "VCCLCompilerTool": {
             "AdditionalUsingDirectories": [
               "%ProgramFiles(x86)%/Microsoft Visual Studio 14.0/VC/lib/store/references",
-              "%ProgramFiles%/Microsoft Visual Studio 14.0/VC/lib/store/references"
+              "%ProgramFiles%/Microsoft Visual Studio 14.0/VC/lib/store/references",
+              "%ProgramFiles(x86)%/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.24.28314/lib/x86/store/references",
+              "%VCToolsInstallDir%/lib/x86/store/references"
```

Comment:
```suggestion
              "$(VCToolsInstallDir)lib/x86/store/references"
```
I guess this will make build system resolve path, instead of depending on preconfigured environment.